ci-operator/step-registry/openshift-extended/upgrade/prenonNoc/openshift-extended-upgrade-prenonNoc-ref.yaml

```diff
@@ -0,0 +1,65 @@
+ref:
+  as: openshift-extended-upgrade-prenonNoc
```

Comment:
The content of this step is the same as `openshift-extended-upgrade-pre`, the only difference is `cli: oc4pre`, right?

BTW, I wonder what means `nonNoc` ?

Reply:
yes.
nonNoc means non N oc
I can not make good step name, could you please provide one? thanks

Reply:
How about naming it `openshift-extended-upgrade-pre-custom-cli`, once that, maybe we also need to change `cli: oc4pre` to `cli: custom`

Reply:
ok, done